test/services/claimToAdjust/ClaimToAdjustServiceSpec.scala

```diff
@@ -235,4 +241,206 @@ class ClaimToAdjustServiceSpec extends TestSupport with MockFinancialDetailsConn
     }
   }
 
+  "getAmendablePoaViewModel" should {
+    "return a future right with AmendablePoaViewModel when POAs are unpaid" in {
+
+      val taxYear:Int = 2023
+
+      val outstandingAmount:BigDecimal = 1250.0
+
+      val chargeHistories: List[ChargeHistoryModel] = List(
+        ChargeHistoryModel(s"$taxYear", "1040000124", LocalDate.of(taxYear, 2, 14), "ITSA- POA 1", 2500, LocalDate.of(taxYear + 1, 2, 14), "Customer Request", Some("001")),
+        ChargeHistoryModel(s"$taxYear", "1040000125", LocalDate.of(taxYear, 2, 14), "ITSA - POA 2", 2500, LocalDate.of(taxYear + 1, 2, 14), "Customer Request", Some("002")))
+
+      setupGetCalculationList(testNino, "22-23")(calculationListSuccessResponseModelNonCrystallised)
+
+      setupGetChargeHistory(testNino, Some("ABCD1234"))(ChargesHistoryModel(
+        idType = "NINO",
+        idValue = testNino,
+        regimeType = "ITSA",
+        chargeHistoryDetails = Some(chargeHistories)
+      ))
+
+      setupMockGetFinancialDetails(taxYear, testNino)(financialDetailsModelBothPoAsWithOutstandingAmount(taxYear, outstandingAmount))
+
+      val f = fixture(LocalDate.of(taxYear, 8, 27))
+      val result = f.testClaimToAdjustService.getAmendablePoaViewModel(testUserNino)
+
+      whenReady(result) {
```

Comment:
We are using option 2 across the projects, this comment is code style specific, so really optional and we can merge this PR assume we can tidy up these tested later or in another PR:
=> "The three ways this trait enables you to test futures are:"
https://www.scalatest.org/scaladoc/3.2.7/org/scalatest/concurrent/Futures.html